src/Pages/LandingPage.tsx

```diff
@@ -2,13 +2,45 @@ import { ExerciseListElement } from '../Components/LandingPage/ExerciseListEleme
 import { ScrollableList } from '../Components/Shared/ScrollableList';
 export const LandingPage = () => {
   const listView = [
-    <ExerciseListElement exercise="a" />,
-    <ExerciseListElement exercise="b" />,
+    <ExerciseListElement
+      exercise="Beginner Transitions"
```

Comment:
is ``exercise="Beginner Transitions"`` the title of the exercise? If so then i think the prop (?) should be named ``title``